### PR TITLE
Travis support

### PR DESCRIPTION
**Pull request by dionyziz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: python
+sudo: true
+before_script:
+    - pip install pep8
+    - npm install jshint
+    - cd backend && pip install -r requirements.txt && cd ..
```
Comment (dimkarakostas) [inline .travis.yml]: TODO: same for sniffer

**Comment by dimkarakostas** [comment]
LGTM

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+language: python
+sudo: true
+before_script:
```
Comment (dionyziz) [inline .travis.yml]: done

File: .travis.yml (hunk 1 above)
Comment (dionyziz) [inline .travis.yml]: done

**Pull request merged**